src/lib/support/CommonPersistentData.h

```diff
@@ -0,0 +1,92 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *          Contains a class handling creation of linked list of stored persistent data.
+ */
+
+#include <lib/core/DataModelTypes.h>
+#include <lib/support/PersistentData.h>
+
+namespace chip {
+namespace CommonPersistentData {
+
+constexpr size_t kPersistentFabricBufferMax = 32;
+constexpr uint8_t kdefaultUndefinedId       = 0;
```

Comment:
```suggestion
constexpr uint8_t kDefaultUndefinedIndex       = 0;
```

Reply:
The naming here is inconsistent. The constant is and ID, the type is an "index", but the field is an "entry". I don't mind which, but you should pick one. For instance:
```
constexpr uint8_t kDefaultUndefinedIndex       = 0; 
...
 IndexType first_index = kDefaultUndefinedIndex;
```

Reply:
Switched everything to "entry" in latest commit